libs/utils/env.py

```diff
@@ -600,6 +601,14 @@ def _init_target_gem5(self):
             system['platform']['description']))[0]
         self.conf['board'] = self.conf['board'].lower() + suffix
 
+        energy_file = self.conf['board'] + '_energy';
+        try:
+            self.nrg_model = globals()[energy_file]
```

Comment:
This patch is a RFC at the moment as I'm not sure if there's a better way to select a specific energy model object, dynamically, based on board and platform information in the target configuration.